cmd/kubeadm/app/util/config/testdata/conversion/node/internal.yaml

```diff
@@ -2,18 +2,19 @@ AdvertiseAddress: 192.168.2.2
 CACertPath: /etc/kubernetes/pki/ca.crt
 ClusterName: kubernetes
 ControlPlane: false
-DiscoveryFile: ""
-DiscoveryTimeout: 5m0s
-DiscoveryToken: abcdef.0123456789abcdef
-DiscoveryTokenAPIServers:
-- kube-apiserver:6443
-DiscoveryTokenCACertHashes: null
-DiscoveryTokenUnsafeSkipCAVerification: true
+Discovery:
```

Comment:
This will break old configurations.

For coming `v1beta1`, we cound use a new section like this.

But for current `v1alpha3`, shall we use below instead in `types.go`,
```go
Discovery Discovery `json:",inline"`
```

Reply:
Actually, it won't, since `v1alpha3` is also unreleased and is massively different than `v1alpha2` of 1.11.
I am also not a fan of inlining it, because that way the context of the options in `Discovery` is gone.